packages/angular/build/src/builders/application/options.ts

```diff
@@ -706,3 +707,22 @@ function normalizeExternals(value: string[] | undefined): string[] | undefined {
 
   return [...new Set(value.map((d) => (d.endsWith('/*') ? d.slice(0, -2) : d)))];
 }
+
+async function findFrameworkVersion(projectRoot: string): Promise<string> {
```

Comment:
Any reason why not using the `VERSION` const exported from `@angular/core` directly?